### ISSUE TITLE
Pending Pacts - pactbroker.providerBranches not working

### ISSUE THREAD

**Issue by praveen-em** [pact-foundation/pact-jvm#1562]
I tried using `pactbroker.providerBranches` (replacing `pactbroker.providerTags`) on my provider with latest release `4.3.8`.  It is not able to fetch pending and wip pacts using branches. It was able to fetch the pending and wip pacts using tags. Doing bit of investigation, it appears the problem is because [this code](https://github.com/pact-foundation/pact-jvm/blob/5917d51694b40df834f2f176e8b1f97d5e868e48/core/pactbroker/src/main/kotlin/au/com/dius/pact/core/pactbroker/PactBrokerClient.kt#:~:text=body%5B%22providerVersionBranches%22%5D%20%3D%20jsonArray(providerBranches)) populates `providerVersionBranches` but Pact broker expects `providerVersionBranch` I think (@bethesque ?). This is my suspect but I am not sure. 

Pactbroker log: 
```
2022-05-26 17:48:01.586098 I [7:puma srv tp 001] PactBroker::Api::Resources::ProviderPactsForVerification -- Fetching pacts for verification by pactdemo-provider-springboot -- {:provider_name=>"pactdemo-provider-springboot", :params=>{"consumerVersionSelectors"=>[{"mainBranch"=>true}, {"deployedOrReleased"=>true}], "includePendingStatus"=>true, "includeWipPactsSince"=>"2022-05-01", "providerVersionBranches"=>["mybranch"], "providerVersionTags"=>[]}}
```

**Comment by YOU54F** [comment]
Hi,

I think you are correct

https://github.com/pact-foundation/pact-ruby/blob/f0e37a4794e72d0a469298ff592d69f1304c9022/lib/pact/pact_broker/fetch_pact_uris_for_verification.rb#L82-L83

Would happily accept a pull request if you have time?

**Comment by praveen-em** [comment]
sure, I will git it a shot.

**Comment by praveen-em** [comment]
@YOU54F , submitted PR https://github.com/pact-foundation/pact-jvm/pull/1565

**Comment by bethesque** [comment]
The API is documented here https://github.com/pact-foundation/pact_broker/blob/master/lib/pact_broker/doc/views/provider-pacts-for-verification.markdown

**Comment by praveen-em** [comment]
Fixed in release 4.3.9

**Comment by TGNThump** [comment]
My bad 😅
